.gitignore

```diff
@@ -7,6 +7,8 @@ pids
 *.pid
 *.seed
 
+.DS_Store
```

Comment:
can we keep this out of the gitignore here? i've generally been using a global gitignore for this sort of thing